services/net/notification/Models/TemplateModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿using TNO.API.Areas.Services.Models.Content;
+
+namespace TNO.Services.Notification.Models
+{
+    public class TemplateModel
+    {
+        /// <summary>
+        /// get/set - The key.
+        /// </summary>
+        public string Key { get; set; } = string.Empty;
```

Comment:
This property has nothing to do with the purpose of the model.  Neither do you use it.

Reply:
Removed, thanks